fill in missing package.json fields

rmg opened the pull request:
This will allow 'npm bugs' to report a useful URL and also let a useful
repo link show up on the npmjs.com package page.

Closes #41 

Connect to #41

sam-github commented on package.json:
```diff
@@ -1,6 +1,16 @@
+  "keywords": [
+    "strongloop",
+    "mesh"
+  ],
```
We don't need keywords, because no one needs to search for this package and find it.

sam-github replied:
and if they are there, someone might start asking that they be maintained... like changing strongloop to IBM, so better not to have, IMO

sam-github commented on package.json:
```diff
@@ -1,6 +1,16 @@
+  "bugs": {
+    "url": "https://github.com/strongloop/strong-mesh-client/issues"
+  },
+  "author": "StrongLoop <callback@strongloop.com>",
```
I mostly see `engineering@strongloop.com` for this in our other packages, though I wouldn't claim that is consistent.

sam-github commented on package.json:
```diff
@@ -1,6 +1,16 @@
+    "url": "https://github.com/strongloop/strong-mesh-client/issues"
+  },
+  "author": "StrongLoop <callback@strongloop.com>",
+  "readmeFilename": "README.md",
```
I thought that was default for some reason, do I need to start adding this field?

sam-github commented:
Capitalize the commit message.

Otherwise looks good to me, aside from the nits.

rmg commented:
Addressed nits. They were mostly just from running `npm init`. For the email address I agree that engineering makes more sense.. I just assumed it was internal only. Not a big deal either way, so I changed it.
